backend/engine/plugins/github_repo_health/cli/src/rules/branch_protection_commit_signing.py

```diff
@@ -26,7 +26,7 @@ def check(github, owner, repo, branch, config={}):
         try:
             protection_config = github.get_branch_protection(owner, repo, branch)
         except GithubException as e:
-            return add_metadata(False, BranchCommitSigning, config, error_message=e.data.get("message"))
+            return add_metadata(False, BranchProtectionCommitSigning, config, error_message=error_message)
```

Comment:
Does `error_message` get defined someplace? My IDE claims it's not, and I noticed the other rules still seem to use `e.data.get("message")` here

Reply:
Ah good catch, had added a bit to handle the case where PyGithub raises a `Branch not protected` error to fail without error, but removed it when I realized it didn't really need to be handled specially and missed this.